domador.js

```diff
@@ -150,7 +150,8 @@ Domador.prototype.append = function append (text) {
 };
 
 Domador.prototype.br = function br () {
-  this.append('  ' +  this.left);
+  if (!this.inTable) this.append('  ' +  this.left);
+  else this.append('  <br />');
```

Comment:
I'd rather do all of them in a single place. Not concerned about `<br>` vs `<br/>`.